src/compute/src/compute_state.rs

```diff
@@ -157,13 +159,20 @@ impl<'a, A: Allocate> ActiveComputeState<'a, A> {
     fn handle_update_configuration(&mut self, params: ComputeParameters) {
         info!("Applying configuration update: {params:?}");
 
-        if let Some(v) = params.max_result_size {
+        let ComputeParameters {
+            max_result_size,
+            dataflow_max_inflight_bytes,
+            persist,
+        } = params;
+
+        if let Some(v) = max_result_size {
             self.compute_state.max_result_size = v;
         }
+        if let Some(v) = dataflow_max_inflight_bytes {
+            self.compute_state.dataflow_max_inflight_bytes = v;
+        }
```

Comment:
Only tangential to your change: This structure of setting parameters this way doesn't allow us to reset them to the built-in value within compute. This can be a problem in the future and we should think about alternatives.

Reply:
This is true. Although it currently works out fine when you reset a variable in SQL: Adapter will just generate a new config where all values are `Some` with the defaults filled in. This will stop working once we introduce our first optional config value, unless we want to make it an `Option<Option<T>>` in the `ComputeParams` struct.

Reply:
https://github.com/MaterializeInc/materialize/issues/17650